src/windows/system.rs

```diff
@@ -239,7 +240,6 @@ impl SystemExt for System {
                 if ntstatus == STATUS_INFO_LENGTH_MISMATCH {
                     // GetNewBufferSize
                     if cb_needed == 0 {
-                        process_information.reserve(buffer_size);
```

Comment:
Putting the reserve call here failed to reserve if cb_needed was >0

Reply:
This also would do nothing because the Vec already had capacity for buffer_size additional elements, it hasn't been doubled yet